test/scapy/layers/bluetooth.uts

```diff
@@ -364,4 +380,4 @@ r = raw(SM_Hdr()/SM_DHKey_Check(dhkey_check="scapy"))
 assert r == b'\rscapy\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
 
 p = SM_Hdr(r)
-assert SM_DHKey_Check in p and p.dhkey_check[:5] == b"scapy"
+assert SM_DHKey_Check in p and p.dhkey_check[:5] == b"scapy"
```

Comment:
Please remove this change.

Reply:
I add new line at the end of the file!

Check this please.

Reply:
@guedou 
An error appeared in 'continuous-integration/appveyor/pr'.
I think it isn't related to my PR,,
can you check this please?

Thanks.

Reply:
@guedou  I checked the details of failed test, and I think those errors are not related to Bluetooth files...
can you check it please?

Reply:
> Please remove this change.

@guedou I checked the details of failed test, and I think those errors are not related to Bluetooth files...
can you check it please?

thanks!